test/parallel/test-fs-promisified.js

```diff
@@ -29,3 +30,5 @@ common.refreshTmpDir();
     fs.closeSync(fd);
   }));
 }
+
+exists(__filename).then((x) => assert(x));
```

Comment:
I’d recommend wrapping these in `{}`, too (even if it’s just a single line), and the callback should probably be wrapped in `common.mustCall()`